ImageGallery/src/org/sleuthkit/autopsy/imagegallery/actions/OpenAction.java

```diff
@@ -153,10 +156,20 @@ public void performAction() {
         }
         Platform.runLater(() -> {
             ImageGalleryController controller;
-            try {
-                controller = ImageGalleryModule.getController();
```

Comment:
ImageGalleryModule now unused import.

Also copyright update

Reply:
Updated / corrected copyright and removed unused import.